website/docs/components/copy/button/partials/code/component-api.md

```diff
@@ -0,0 +1,25 @@
+## Component API
+
+This component uses [ember-cli-clipboard](https://github.com/jkusa/ember-cli-clipboard) under the hood.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="size" @type="enum" @values={{array "small" "medium" }} @default="medium"/>
+    <C.Property @name="isIconOnly" @type="boolean" @default="false">
+    Indicates if the button will only contain an icon. An internal check is in place to ensure that accessible text is still applied to the component.
+  </C.Property>
+  <C.Property @name="isFullWidth" @type="boolean" @default="false">
+    Indicates that a button should take up the full width of the parent container.
+  </C.Property>
+  <C.Property @name="text" @required="true" @type="string">
+    The text of the button or value of `aria-label` if `isIconOnly` is set to `true`. If no text value is defined, an error will be thrown.
+  </C.Property>
+  <C.Property @name="textToCopy" @type="string | function">
+    String value or action that returns a string to be copied.
+  </C.Property>
+  <C.Property @name="targetToCopy" @type="string | function">
+     Selector string of element or action that returns an element from which to copy text.
+  </C.Property>
+  <C.Property @name="...attributes">
```

Comment:
it may have gotten lost on the other PR, but `container` is not documented here. Should it?

Reply:
It's been added 👍

Reply:
Do we typically also add this to the "how to use" section or no?

Reply:
Or is that kinda hard in this case since it relies on something like a modal?